### PR TITLE
Resolve name conflict with Atlas

### PR DESCRIPTION
**Pull request by SevenSpheres**
There are two objects named Atlas; a star and a moon. I renamed the moon to "Atlas (moon)".

Maybe I should just update all the data files, since Askaniy seems to have disappeared...

### CODE REVIEW

File: data/dwarfplanets.ssc [hunk 1]
```diff
@@ -37,8 +37,6 @@ ReferencePoint "Pluto-Charon" "Sol"
 {
     CustomOrbit "pluto"
 
-    OrbitColor  [ 0.556863 0.235294 0.576471 ]
```
Comment (375gnu) [inline data/dwarfplanets.ssc]: why did you remove this color?

Comment (SevenSpheres) [reply data/dwarfplanets.ssc]: Since the orbit color for dwarf planets is now specified in the source code, that line is no longer needed

Comment (SevenSpheres) [reply data/dwarfplanets.ssc]: Added it back (I can use rebase now!)

**Comment by pirogronian** [comment]
I think this shpuld be resolved by namespace. Altlas the star is accessible from main namespace, while Atlas the moon is accessible from Saturn namespace, I mean to find from console we have to enter "Saturn/" first. So, technically, there is no reason for change.

**Comment by SevenSpheres** [comment]
> I mean to find from console we have to enter "Saturn/" first.

Yes, this is _because_ there is a star with the same name.

**Comment by 375gnu** [comment]
Pirogronian pointed a better way, so I agree, no need to change.
